### ISSUE TITLE
ConsecutiveComments linter "good" example appears to have incorrect indenting

### ISSUE THREAD

**Issue by anselmbradford** [sds/haml-lint#43]
The ["good" example](https://github.com/causes/haml-lint/blob/master/lib/haml_lint/linter/README.md#consecutivecomments) for the ConsecutiveComments linter has 3 spaces for all lines after the first. In practice this doesn't appear possible to do. Take the following partial snippet:

``` ruby
-# The email partial displays the email icon and address.
   e.g. [icon] admin@example.com

- if email.present?
  %section.email
    …
```

With 3 spaces present for the second line of the comment, the page fails with a `Haml::SyntaxError` error  of `Inconsistent indentation: 2 spaces used for indentation, but the rest of the document was indented using 3 spaces.`.

A correctly functioning example instead uses 2 spaces for the lines after the first:

``` ruby
-# The email partial displays the email icon and address.
  e.g. [icon] admin@example.com

- if email.present?
  %section.email
    …
```

Is there a recommended format that left-aligns all pertinent comment lines, something like the following for example?

``` ruby
-# |
  The email partial displays the email icon and address.
  e.g. [icon] admin@example.com
```
